draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -131,36 +131,27 @@
 
 <middle>
 <section title="Introduction" anchor="introduction">
+<section title="Purpose" anchor="intro.purpose">
 <t>
-   The Hypertext Transfer Protocol (HTTP) is a stateless application-level
-   request/response protocol that uses extensible semantics and
-   self-descriptive messages for flexible interaction with network-based
-   hypertext information systems. HTTP is defined by a series of documents
-   that collectively form the HTTP/1.1 specification:
+   The Hypertext Transfer Protocol (HTTP) is a family of stateless,
+   application-level, request/response protocols that use a generic interface,
```

Comment:
I think "share" would be better than "use" here; that better reflects the relationship between the various "family members."